src/common/ast-path.js

```diff
@@ -110,6 +110,21 @@ class AstPath {
     return result;
   }
 
+  /**
+   * @param {() => void} callback
+   * @internal Unstable API. Don't use in plugins for now.
+   */
+  try(callback) {
+    const { stack } = this;
+    const stackBackup = [...stack];
+    try {
+      callback();
```

Comment:
```suggestion
      return callback();
```

Let's add `return` here, `path.try(() => path.map(...))`